90zfsbootmenu/module-setup.sh

```diff
@@ -79,27 +79,17 @@ install() {
   dracut_install /usr/lib/udev/vdev_id
   dracut_install /usr/lib/udev/zvol_id
   dracut_install tac
-  dracut_install awk
   dracut_install basename
-  dracut_install cut
   dracut_install head
   dracut_install kexec
   dracut_install fzf
   dracut_install mktemp
   dracut_install sort
   dracut_install sed
-  dracut_install grep
```

Comment:
I've saved myself a few times using `grep` with context in place of a real pager, so I'd advocate keeping it in here. That being said, `less` appears to be in my boot image and installed by `99base`, so I'm not sure why I couldn't access it from the emergency shell.

Reply:
That's reasonable - I've added it back in.